docs/uninstall-olm.md

```diff
@@ -0,0 +1,53 @@
+# How do I uninstall OLM?
+
+You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
+
+```bash
+export olm_release=0.13.0
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/olm.yaml
+```
+
+> NOTE: You can identify which version of OLM you are using by inspecting the version of the packageserver CSV.
+
+> ```bash
+> kubectl -n <OLM namespace, e.g.: olm, etc> get csvs
```

Comment:
This is a pattern I haven;'t seen before. I would recommend simplifying things like so:
```bash
$ export OLM_NAMESPACE=<olm-namespace>
$ kubectl -n $OLM_NAMESPACE get csvs
```

Reply:
+1